dags/mozaggregator_prerelease.py

```diff
@@ -124,9 +124,8 @@
 
 trim_database = gke_command(
     task_id="trim_database",
-    cmds=["bash"],
+    cmds=["python"],
```

Comment:
Was this failing with `can't execute binary file`?

Reply:
Indeed, it was.